src/test/java/com/yahoo/sketches/theta/ConcurrentThetaEstimationTest.java

```diff
@@ -0,0 +1,122 @@
+package com.yahoo.sketches.theta;
+
+import static com.yahoo.sketches.Util.DEFAULT_UPDATE_SEED;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Test;
+
+import com.yahoo.memory.WritableMemory;
+
+/**
+ * @author eshcar
+ * @deprecated  This approach is not correct.  Please refer to the characterization repository.
+ */
+@Deprecated
+public class ConcurrentThetaEstimationTest {
```

Comment:
Since the concurrent code has not yet been released, there is no point in keeping deprecated code around.  Please remove.

Reply:
I am keeping this code until I have the multi-threaded tests in characterization module, Then I will remove it. This is the next step in the plan